ph5/clients/tests/test_ph5tostationxml.py

```diff
@@ -415,14 +562,58 @@ def test_create_obs_network(self):
         with LogCapture() as log:
             log.setLevel(logging.WARNING)
             ret = self.parser.create_obs_network()
-            self.errmsgs.append('All response file names are blank in response'
-                                ' table. Check if resp_load has been run.')
             self.assertEqual(set(rec.msg for rec in log.records),
                              set(self.errmsgs + self.warnmsgs))
             self.assertEqual(ret.code, 'AA')
             self.assertEqual(len(ret), 1)
             self.assertEqual(ret.stations[0].code, '1115')
 
 
+class TestPH5toStationXML_Response_NI_MISMATCH(LogTestCase, TempDirTestCase):
+    def test_Response_NI_MISMATCH(self):
+        # Uncomment line 401 and comment line 402 to prove test
+        # datapath = '../../test_data/ph5/'
+        datapath = os.path.join(self.home,
+                                'ph5/test_data/ph5/response_table_n_i')
+        self.ph5_path_eror = os.path.join(self.home,
+                                          datapath)
+        self.ph5sxml, self.mng, self.parser = getParser(self.ph5_path_eror,
+                                                        "master.ph5",
+                                                        "RESPONSE")
+        self.parser.add_ph5_stationids()
+        station = self.parser.read_stations()
+        self.assertEqual(len(station[0].channels[0].
+                         response._get_overall_sensitivity_and_gain()), 2)
+        try:
+            station[0].channels[1].response._get_overall_sensitivity_and_gain()
+        except IndexError:
+            self.mng.ph5.close()
+            self.fail("Response information is not present"
+                      " in output stationxml.")
+        self.mng.ph5.close()
+
+
+class TestPH5toStationXML_Response_NI_DUPLICATE(LogTestCase, TempDirTestCase):
+    def test_Response_NI_DUPLICATION(self):
+
+        datapath = os.path.join(self.home,
+                                'ph5/test_data/ph5/response_table_n_i_dup')
+        self.ph5_path_eror = os.path.join(self.home,
+                                          datapath)
+        self.ph5sxml, self.mng, self.parser = getParser(self.ph5_path_eror,
+                                                        "master.ph5",
+                                                        "RESPONSE")
```

Comment:
Flag --stationxml_on_error can be set by passing argument `stationxml_on_error=True/False`